framework/src/postprocessors/SideIntegralVariablePostprocessor.C

```diff
@@ -48,6 +48,8 @@ SideIntegralVariablePostprocessor::computeQpIntegral()
   {
     const FaceInfo * const fi = _mesh.faceInfo(_current_elem, _current_side);
     mooseAssert(fi, "We should have an fi");
+    mooseAssert(fi->isBoundary(),
+                "SideIntegralVariablePostprocessor should only be used at boundaries");
```

Comment:
Hmm I don't really agree with this assertion. I think we should be able to run this on internal mesh sidesets as long as those sidesets represent a boundary for where that variable lives.

Reply:
ok deleting, getBoundaryFaceValue will catch the issue if not used at a boundary for that variable anyway.

Rewording it to be more friendly since it becomes a user-facing assert